runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/PTransformTranslation.java

```diff
@@ -292,6 +293,13 @@ public boolean canTranslate(PTransform<?, ?> pTransform) {
     private static final Map<Class<? extends PTransform>, TransformPayloadTranslator>
         KNOWN_PAYLOAD_TRANSLATORS = loadTransformPayloadTranslators();
 
+    private static List<String> knownSDKTransforms =
```

Comment:
Renamed to sdkTransformsWithEnvironment since we are trying to maintain current behavior in this PR. In the future I think we should set environment ID in all transforms by default and let runners override for naively implemented transforms. Created https://issues.apache.org/jira/browse/BEAM-9001 for this.